decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -63,16 +67,19 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
                                      y: self.view.frame.origin.y,
                                      width: self.view.frame.width,
                                      height: window.origin.y + window.height - keyboardSize.height)
+            self.tableView.contentInset = UIEdgeInsets(top: 0,left: 0,bottom: 10,right: 0)
```

Comment:
- Add an empty line space before this, code looks too clogged.
- Add a comment to show why this is necessary.
- Use whitespace to separate the function call parameters.